src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -156,6 +156,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('acr replication create') as c:
         c.argument('replication_name', help='The name of the replication. Default to the location name.', completer=None)
 
+    for scope in ['acr replication create', 'acr replication update']:
+        help_str = "Allow routing to this replication. Requests will not be routed to a disabled replication." \
+                   " Data syncing will continue regardless of the region endpoint status."
+        help_str += ' Default: true.' if 'create' in scope else ''  # suffix help with default if command is for create
+
+        with self.argument_context(scope) as c:
+            c.argument('region_endpoint_enabled', arg_type=get_three_state_flag(), help=help_str, is_preview=True)
```

Comment:
Can we actually preview a flag?

Reply:
Yes we can, it looks like this: 

![image](https://user-images.githubusercontent.com/15149427/83068189-c43bc000-a01c-11ea-81f7-841e2c4f33ea.png)

Reply:
@djyou admin_enabled defaults to false, however, false defaults aren't surfaced in help text. public access enabled's default is left to the sdk if None. Sdk defaults to None.

Our sdk does default to True for --region-endpoint-enabled. I added the logic I had to ensure it is always consistent in behavior. But I guess that's overkill. Should I leave default to sdk but document that region endpoint will always be true (assuming it would be breaking change to change this)

See here for acr_create defaults: https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acr/custom.py#L35-L47

Reply:
You can add `default='true'` or  `default='false'` in `c.argument()`.